lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -46,8 +46,7 @@ use List::Util 'shuffle';
 use OpenQA::IPC;
 use sigtrap handler => \&normal_signals_handler, 'normal-signals';
 use OpenQA::Scheduler;
-
-
+use if !$Test::{'Compile::'}, 'OpenQA::WebSockets::Server';
```

Comment:
@foursixnine @coolo @mudler 

I need opinion in this piece.

Basically the Test::Compile fails because of circular dependencies so i came up with this... But i want to hear from you guys :-)

Reply:
Just don't create circual dependencies - as easy.

Reply:
Without that change the Test::Compile fails when it tries to compile the JobModules.pm. That code is also perfectly valid and it breaks the circular. Please note that only the Test::Compile has problems when you try to run the code. 

Since i only need a constant i see the possible solutions:
1- either i move the constant out of there to another file (It's the websocket interface api, so should be there, no?)
2- one of these: stop using constants, no strict 'subs', use AUTOLOAD to import the method when the invocation of the constant fails (some of these might not work. I'm just trying to remember of possible solutions).

Reply:
It does because it's using schemas - i would rather go for 1)

Reply:
@mudler @coolo @foursixnine 

That looks like the best "non-hackish" solution. So if everybody agrees I'll move the interface constants to a new lib/OpenQA/Constants.pm file.